res/css/views/rooms/_EventBubbleTile.pcss

```diff
@@ -573,6 +586,17 @@ limitations under the License.
     }
 }
 
+.mx_EventTile.mx_EventTile_bubbleContainer[data-layout="bubble"],
+.mx_EventTile.mx_EventTile_leftAlignedBubble[data-layout="bubble"],
+.mx_GenericEventListSummary[data-layout="bubble"][data-expanded="false"] {
+    .mx_EventTile_line {
+        > a, /* timestamp wrapper anchor */
+        .mx_MessageActionBar + .mx_MessageTimestamp {
+            left: -77px;
```

Comment:
Here it might be better to use a variable instead for better spacing management, with simplifying selectors.

Reply:
I don't understand why this had to move out of the block below - can you explain, please?

Reply:
The block below includes `mx_EventTile.mx_EventTile_info[data-layout="bubble"]`. Until it is confirmed that `left: -70px` can be applied to all of them, it should be safe to let `left: -77px` applied to the three of them.